pyap_beauhurst/source_GB/data.py

```diff
@@ -351,31 +351,6 @@
 
 postal_code = r"""
 (?P<postal_code>
```

Comment:
would it be worth to have these as optional?

Reply:
I think probably not at this point - the pattern is quite different to UK postcodes, and I think any positive of matching things in these postcodes would be far outweighed then by the negative of the false positives it might bring in. 

If we decide to look at this in the future, we have it here in the git history?